src/hotspot/share/opto/superword.cpp

```diff
@@ -2346,6 +2348,204 @@ void SuperWord::verify_packs() {
 }
 #endif
 
+class PacksetGraph {
```

Comment:
May short description of what this class is doing. Something similar to your comment for `schedule()`.